fulcrumapp/fulcrum-query-sql
# Bugfix/flcrm 10359 fulcrum core is causing problems

# Description
Fulcrum core has a location field that has `address` `latitude` and `longitude` part after the key so I made sure to add them this can be tested with `fulcrum-core@0.16.0` in the `fulcrum` app

Testing

- change your package.json in `fulcrum-components` with `fulcrum-core` to `0.16.0` and `fulcrum-query-sql` to `github:fulcrumapp/fulcrum-query-sql#bugfix/FLCRM-10359-fulcrum-core-is-causing-problems`
- remove the yarn.lock and run `yarn install` again
- run `skaffold dev`
- open your app and the tasks page should be working fine
